src/client/static/analytics/ophan.ts

```diff
@@ -22,7 +22,7 @@ const addRefToOphanFollow = () => {
 
 addRefToOphanFollow();
 
-import 'ophan-tracker-js';
+import '@guardian/ophan-tracker-js';
```

Comment:
It looks like we still using record from the older import `import { record } from '@/client/lib/ophan';` at the top ... could we do something like this instead:

```
import ophan from '@guardian/ophan-tracker-js';

export const init = () => {
  ophan.record({
```

Reply:
I had a look at this and it didn't seem as straightforward as it looked! Unforunately theres 2 issues that I found:

1. The version of `@guardian/ophan-tracker-js` is tool old to use the newer import format.
2. Upgrading to the newer version of `@guardian/ophan-tracker-js` silently broke tracking and uses a different schema for events than we do. Even after switching to the new imports tracking was still silently broken.